lib/core/src/Cardano/Wallet/DB/Sqlite/Types.hs

```diff
@@ -263,6 +267,33 @@ instance PathPiece TxId where
     toPathPiece = toText . getTxId
     fromPathPiece = fmap TxId . fromTextMaybe
 
+--------------------------------------------------------------------------------
+-- Tokens
+--------------------------------------------------------------------------------
+
+instance PersistField TokenName where
+    toPersistValue = toPersistValue . toText
+    fromPersistValue = fromPersistValueFromText
+
+instance PersistFieldSql TokenName where
+    sqlType _ = sqlType (Proxy @Text)
+
+instance PersistField TokenPolicyId where
+    toPersistValue = toPersistValue . toText
+    fromPersistValue = fromPersistValueFromText
+
+instance PersistFieldSql TokenPolicyId where
+    sqlType _ = sqlType (Proxy @Text)
+
+instance PersistField TokenQuantity where
+    toPersistValue = toPersistValue . toText
+    fromPersistValue = fromPersistValueFromText
```

Comment:
Perhaps add a comment here to mention why we are serializing to text.

(Neither SQLite nor `persistent-sqlite` support a big integer data type.)

Reply:
Fixed in `b76899d31d28b0d6342eea43e28f9e399b00c55b`.